Fix for the overly large gaps between news items on the news index page.

drcongo opened the pull request:
This ensures that we only set the fixed-height class on items on the blog index page, so the news index page doesn't get gaps larger than we want between items.

AdamChrimes commented:
Fixed an issue with missing closing brackets and some missing logic following testing on staging.

AdamChrimes commented:
The cookie banner link hover hasn't been fixed in the `cookie-consent` solution, so I've reverted the commit to remove the hotfix styles.

drcongo committed:
Fix for the overly large gaps between news items on the news index page.

rjw1 committed:
Add the mailcap package

We think that we are not sending the correct mime type for CSS files.
This should help

pezholio committed:
Merge pull request #40 from dxw/add-mailcap-to-prod-dockerfile

Add the mailcap package
